scheduler/cmd/scheduler/main.go

```diff
@@ -223,17 +219,53 @@ func main() {
 		log.Warn("Not running with scheduler local DB")
 	}
 
-	s := schedulerServer.NewSchedulerServer(logger, ss, es, ps, sched, eventHub)
+	// Setup synchroniser
+	var sync synchroniser.Synchroniser
+
+	if namespace == "" {
```

Comment:
Strictly speaking the namespace can be set via a command-line flag, but I think testing it with the empty string is still the right thing to do here?